media/js/src/simulations/simulation2/components/simulationTwo.jsx

```diff
@@ -121,20 +124,22 @@ export const SimulationTwo = () => {
         // If the user has completed the first question
         // for the current dataset, show the second with a
         // general question.
-        const takeaways = checkComplete() > 1 && isComplete['general'] != true ?
+        
```

Comment:
My linter is red at this line for trailing spaces

Reply:
Got it. Strange, my linter didn't pick it up.